File: app/src/main/java/com/neome/java/api/ent/base/dto/DtoEntDeeplinkSpreadsheetEditor.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdLayoutGrid;
import com.neome.java.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DtoEntDeeplinkSpreadsheetEditor extends DtoEntDeeplink
{
  @Nullable
  public MetaIdLayoutGrid layoutSpreadsheetId;

  @Nullable
  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/ent/base/dto/DtoEntDeeplinkSpreadsheetEditor.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.base.dto

import com.neome.api.ent.base.dto.DtoEntDeeplink
import com.neome.api.meta.base.Types.MetaIdLayoutGrid
import com.neome.api.meta.base.Types.MetaIdSpreadsheet

interface DtoEntDeeplinkSpreadsheetEditor : DtoEntDeeplink
{
  val layoutSpreadsheetId: MetaIdLayoutGrid?
  val spreadsheetId: MetaIdSpreadsheet?
}